include/motor-control/core/tasks/motion_controller_task.hpp

```diff
@@ -136,13 +183,55 @@ class MotionControllerMessageHandler {
             can_client.send_can_message(can::ids::NodeId::host, response);
         }
     }
+
     void handle(const can::messages::GetMotorUsageRequest& m) {
         controller.send_usage_data(m.message_index, usage_client);
     }
 
+    void handle(
+        const motor_control_task_messages::RouteMotorDriverInterrupt& m) {
+        if (m.debounce_count > 9) {
```

Comment:
this `9` value and the delay value below should probably be constexprs defined somewhere prominent at the top of the file - they're key values for some behavior and it would be good to find it in a consistent place